src/components/Admin/AlterUsers.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { useSelector, useDispatch, shallowEqual } from 'react-redux'
+import { Table } from '../Table/Table';
+import { useState } from 'react';
+import SelectedUserModal from './SelectedUserModal';
+import { setSelectedAlterUser, deleteUnapprovedUser, setAlterUsers } from '../../store/reducers/adminQuery';
+
+const AlterUsers = ({users, approve}) => {
+
+    const dispatch = useDispatch()
+    const [showModal, setShowModal] = useState(false)
+    const { currentUser, selectedAlterUser } = useSelector(state => ({
+        selectedAlterUser: state.adminQuery.selectedAlterUser,
+        currentUser: state.user
+    }), shallowEqual)
+
+    const openModalAndAlterUser = (user)  => {
+        dispatch(setSelectedAlterUser({...user}))
+        setShowModal(true)
+    }
+
+    const deleteUser = (user) => {
+        if (window.confirm("Are you sure you want to delete this account?")) {
+            dispatch(disableButton())
+            dispatch(deleteUnapprovedUser(user))
```

Comment:
Since we are gonna be using this page as a component for both `Approved` and `Unapproved` users. It'll be best to rename this function with something more generic....maybe `deleteUserData`.

Please find all instances and connection of this dispatch function in the `reducers`, `constants`, `sagaRoot`, `handler` and `request` files and change accordingly as well